test/flint/fq_default-test.jl

```diff
@@ -343,6 +343,7 @@ end
   @test f(ZZ(1)) == R(1)
   x = preimage(f, R(1))
   @test x isa ZZRingElem && x == 1
+  @test krull_dim(R) == 0
```

Comment:
To me this testset is an unmotivated place for testing Krull dimension. It has the purpose of testing residue fields and the canonical epimorphism themselves. Maybe this could be moved to another testset which already tests krull dimension?

Reply:
I agree... when I wrote in my initial comments "Also, these tests should be in their own testset" I did not refer to just the one test the comment was next to, I really meant *all of the new tests for `krull_dim`, in all files*.